README.md

```diff
@@ -1,14 +1,27 @@
 # github-action-test-compare
 
-Compare tests against production
+**Compare tests against production**
 
-## Basic usage
+This GitHub Action automatically runs the tests within a PR against the target branch and adds a warning comment to the PR if the tests pass. This action can be used to determine if tests added or modified actually test production changes within the same PR.
```

Comment:
I would rephrase the last sentence to be a bit straight to the point. This action's intention is to test if the PR is testing the right thing.

Note we should also mention this applies mostly to "fixes prs".